Snowflake.Data/Configuration/EasyLoggingConfigParser.cs

```diff
@@ -61,5 +64,30 @@ private void Validate(ClientConfig config)
                 EasyLoggingLogLevelExtensions.From(config.CommonProps.LogLevel);
             }
         }
+
+        private void CheckForUnknownFields(string fileContent, ClientConfig config)
+        {
+            // Parse the specified config file and get the key-value pairs from the "common" section
+            JObject obj = (JObject)(JObject.Parse(fileContent).First.First);
+            bool isUnknownField = true;
+            foreach (var keyValuePair in obj)
```

Comment:
this code looks very complicated. It could be simplified if you use linq like that:
```
            JObject.Parse(fileContent).Properties()
                .Where( /*paste a function to filter common section*/)
                .SelectMany(property => property.Value)
                .Where( /*paste a function to filter unknown property*/)
                .ToList()
                .ForEach( /*paste a warning function*/);
```

Reply:
Nice, this looks better. I used GetValues() to shorten it but let me know if you prefer Properties() instead